internal/topology/check/compatibility.go

```diff
@@ -110,10 +113,11 @@ func LocalObjectTemplatesAreCompatible(current, desired clusterv1.LocalObjectTem
 func LocalObjectTemplatesAreInSameNamespace(current, desired clusterv1.LocalObjectTemplate, pathPrefix *field.Path) field.ErrorList {
 	var allErrs field.ErrorList
 	if current.Ref.Namespace != desired.Ref.Namespace {
-		allErrs = append(allErrs, field.Forbidden(
+		allErrs = append(allErrs, field.Invalid(
```

Comment:
```suggestion
		allErrs = append(allErrs, field.Forbidden(
```
Should this be Forbidden too like in `ObjectsAreInTheSameNamespace`?

Up to you, feel free to just resolve.